python/shotgun_fields/__init__.py

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) 2016 Shotgun Software Inc.
+#
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Shotgun Software Inc.
+
+from . import shotgun_field_manager
+ShotgunFieldManager = shotgun_field_manager.ShotgunFieldManager
```

Comment:
Any reason why this can't be:

from .shotgun_field_manager import ShotgunFieldManager